EzPC/EzPC/ast.ml

```diff
@@ -218,18 +230,30 @@ let rec typ_to_string (t:typ) :string =
        | UInt64 -> "uint64"
        | Int32  -> "int32"
        | Int64  -> "int64"
+       | Float  -> "float"
        | Bool -> "bool"
      in
      prefix ^ " " ^ bt_str
   | Array (quals, t, e) ->
      let qual_string = if quals |> List.mem Immutable then "const " else "" in
      qual_string ^ typ_to_string t ^ "[" ^ expr_to_string e ^ "]"
 
+let basetype_to_secfloat (t:base_type) :string =
```

Comment:
Should these function in the codegen file, rather than here?

Reply:
Moved.